packages/mdc-rtl/README.md

```diff
@@ -28,76 +26,20 @@ npm install --save @material/rtl
 
 ## Usage
 
-Simply `@import "@material/rtl/mixins";` and start using the mixins. Each mixin is described below.
+### Sass Mixins
 
-### mdc-rtl
+`$base-property` is a base box-model property. e.g. margin / border / padding.
+`$pos` is a horizontal position property, either "left" or "right".
 
-```scss
-@mixin mdc-rtl($root-selector: null)
-```
-
-Creates a rule that will be applied when an MDC-Web component is within the context of an RTL layout.
-
-Usage Example:
-
-```scss
-.mdc-foo {
-  position: absolute;
-  left: 0;
 
-  @include mdc-rtl {
-    left: auto;
-    right: 0;
-  }
+| Mixin | Description |
+| ----------------------------------------------- | - |
+| `mdc-rtl($root-selector)` | Creates a rule that is applied when the root element is within an RTL context |
+| `mdc-rtl-reflexive-box($base-property, $default-direction, $value, $root-selector)` | Applies the value to the `#{$base-property}-#{$default-direction}` property  in a LTR context, and flips the direction in an RTL context. This mixin zeros out the original value in an RTL context. |
+| `mdc-rtl-reflexive-property($base-property, $left-value, $right-value, $root-selector)` | Emits rules that assign `#{$base-property}`-left to `#{left-value}` and `#{base-property}`-right to `#{right-value}` in a LTR context, and vice versa in a RTL context. |
+| `mdc-rtl-reflexive-position($pos, $value, $root-selector)` | Applies the value to the specified position in a LTR context, and flips the direction in an RTL context. |
 
```

Comment:
Pointed well taken, I agree that we shouldn't include transpiled css here. However, the single line document confuse me when to use which mixin class. Could we consider having a paragraph like:

In case for symmetric RTL properties, you can use `mdc-rtl-reflexive-box`. For example, when button have `margin-left: 8px; margin-right: 0px;` to left edge and `margin-left: 0px; margin-right: 8px` to right edge in RTL mode.
```
@include mdc-rtl-reflexive-box('margin', 'left', 8px, '.mdc-foo')
```